include/mysqlpp/udf_wrappers.hpp

```diff
@@ -215,6 +222,53 @@ class generic_udf<ImplType, INT_RESULT>
   }
 };
 
+struct udf_info {
```

Comment:
I really like the idea of extracting registration / unregistration logic into a reusable file.
The only thing I would  like to change is to put this new code into a separate file, say 'mysqlpp/udf_registration.hpp' so that we won't make every UDF wrapper user depend on MySQL services. This new functionality will still be available but only when user explicitly requests it (includes this 'mysqlpp/udf_registration.hpp' file).

Reply:
Done.